packages/config/config/devices/0x019b/4512689.json

```diff
@@ -20,10 +20,23 @@
 		"min": "0.0",
 		"max": "255.255"
 	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Smoke Sensor (Basic Set)",
+			"description": "When the smoke detector is active, it will send Basic Set (0xFF) to this group. When the smoke detector is deactived, it will send Basic Set (0x00).",
+			"maxNodes": 5,
+			"noEndpoint": true
```

Comment:
Does this **need** to be a node association? `noEndpoint` is usually only used for the Lifeline, but there might be exceptions.

Reply:
I don't believe so. Removed.

Reply:
No it does not, removed.